index.js

```diff
@@ -46,7 +47,9 @@ class ReactRouterGA extends React.Component<Props> {
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
     // Initialize Google Analytics
-    window.ga('create', this.props.id, 'auto');
+    window.ga('create', this.props.id, 'auto', { allowLinker: true });
+    window.ga('require', 'linker');
+    window.ga('linker:autoLink', this.props.domains);
```

Comment:
Technically, linker thing should only be done if `domains` contains at least one element.

Reply:
I'd suggest you add the `if` condition in that case.

Reply:
Done.